src/vs/workbench/contrib/terminalContrib/clipboard/test/browser/terminalClipboard.test.ts

```diff
@@ -23,67 +23,99 @@ suite('TerminalClipboard', function () {
 		setup(async () => {
 			instantiationService = store.add(new TestInstantiationService());
 			configurationService = new TestConfigurationService({
-				[TerminalSettingId.EnableMultiLinePasteWarning]: 'auto'
+				[TerminalSettingId.EnableMultiLinePasteWarning]: 'auto',
+				[TerminalSettingId.AllowSmartPaste]: false
 			});
 			instantiationService.stub(IConfigurationService, configurationService);
 			instantiationService.stub(IDialogService, new TestDialogService(undefined, { result: { confirmed: false } }));
 		});
 
-		function setConfigValue(value: unknown) {
+		function setConfigValue(multiLinePaste: unknown, smartPaste: boolean) {
 			configurationService = new TestConfigurationService({
-				[TerminalSettingId.EnableMultiLinePasteWarning]: value
+				[TerminalSettingId.EnableMultiLinePasteWarning]: multiLinePaste,
+				[TerminalSettingId.AllowSmartPaste]: smartPaste
 			});
 			instantiationService.stub(IConfigurationService, configurationService);
 		}
```

Comment:
Since there's another one now, tests would be easier to read if this was an object:

```
values: { enableMultiLinePaste?: unknown, enableSmartPaste?: boolean }
```

They could also default to false?